File: 프로그래머스/0/120834. 외계행성의 나이/외계행성의 나이.java
class Solution {
    public String solution(int age) {
        String[] ageArr = String.valueOf(age).split("");
        String ar = "abcdefghij";
        String answer = "";
        
        for (int i = 0; i < ageArr.length; i++) {
        	answer += ar.charAt(Integer.valueOf(ageArr[i]));
        } 
        return answer;
    }
}

File: 프로그래머스/0/120834. 외계행성의 나이/외계행성의 나이.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

char* solution(int age) {
    char* answer = (char*)malloc(5 * sizeof(char*));
    char* str = "abcdefghij";
    int a = age;
    int cnt = -1;
    
    while(a > 0){
        a/= 10;
        cnt++;
    }
    
    for(int i=cnt;i>=0;i--){
        answer[i] = str[age%10];
        age /= 10;
    }
    
    return answer;
}